arcsi/api/utils.py

```diff
@@ -159,6 +159,8 @@ def get_shows():
     return shows
 
 def item_duplications_number(item):
-    name_occurrence = int(db.session.query(db.func.count()).filter(Item.name == item.name, Item.number == item.number).scalar())
+    show_items = Show.query.filter(Show.id == item.shows[0].id).first().items
+    show_items_with_same_name = show_items.filter_by(name=item.name)
```

Comment:
Thanks for breaking this long line up into smaller parts!

One naming nitpick. We are in the scope of this function there is no need to add the `show_` prefix. Would you think otherwise? 

Also it is a function that can be read on one screen so it is readable that it is a Show model's query filter.